### PR TITLE
build: Release v38.0

### PR DESCRIPTION
**Pull request by likebreath**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -318,6 +327,75 @@
...
+project](https://github.com/orgs/cloud-hypervisor/projects/6) as iteration
+v38.0. The following user visible changes have been made:
+
+### Group Rate Limiter on Block Devices
+
+Users now can throttle a group of block devices with the new
+`--rate-limiter-group` option. Details can be found from the [I/O
+Throttling documentation](docs/io_throttling.md)
+
+### Thread Pinning Support for Block Device Virt-queue
```
Comment (rbradford) [inline release-notes.md:342]: I think that removing the virt-queue terminology would be better here and just talk about CPU pinning the block device worker thread.

File: release-notes.md [hunk 2]
```diff
@@ -318,6 +327,75 @@
...
+
+### New 'debug-console' Device
+
+A 'debug-console' device is added to provide a user-configurable debug
+port for logging guest information. Details can be found from the [Debug
+IO Ports documentation](docs/debug-port.md).
+
+### Improved VFIO Device Support
+
+all non-emulated MMIO regions of VFIO devices are NOW mapped to the VFIO
```
Comment (rbradford) [inline release-notes.md:360]: Missing a capital letter?

Comment (rbradford) [reply release-notes.md:360]: And why is now full caps?

File: release-notes.md [hunk 3]
```diff
@@ -318,6 +327,75 @@
...
+### Extended CPU Affinity Support
+
+Users now can set the vcpu affinity ot a host CPU with index larger
+than 255.
+
+### Notable Bug Fixes
+
+* Enable HTT flag to avoid crashing cpu topology enumeration software
+such as hwloc in the guest (#6146)
+* Fix several security advisories from dependent crate (#6134, #6141)
```
Comment (rbradford) [inline release-notes.md:375]: s/dependent crate/dependencies/

**Comment by likebreath** [comment]
@rbradford Thanks for the review and catching my typos. All comments are addressed. PTAL.

**Review by rbradford — APPROVED** [review]
You need to update for the PCI bar alignment fix that just merged

**Pull request merged**